tests/components/charts.test.ts

```diff
@@ -597,6 +599,216 @@ describe("charts", () => {
     }
   );
 
+  test.each(["line", "bar", "pie"])(
+    "chart dataseries selectionInput adapts to sheet edition",
+    async (chartType: string) => {
+      createChart(model, {
+        dataSets: ["B1:B10"],
+        labelRange: "A1:A10",
+        type: chartType as "line" | "bar" | "pie",
+      });
+      await nextTick();
+
+      const figures = fixture.querySelectorAll(".o-figure");
+      await simulateClick(figures[0] as HTMLElement);
+      await simulateClick(".o-figure-menu-item");
+      await simulateClick(".o-menu div[data-name='edit']");
+      await nextTick();
+
+      const dataSeries = fixture.querySelectorAll(
+        ".o-sidePanel .o-sidePanelBody .o-chart .o-data-series"
+      )[0];
```

Comment:
you could create directly dataSeriesInput to avoid querySelector("input") every time

same in the other tests